commerce-theme-minium/commerce-theme-minium/src/css/_login.scss

```diff
@@ -60,7 +60,7 @@
         }
     }
 
-    #_com_liferay_login_web_portlet_LoginPortlet_loginForm {
+    #_com_liferay_login_web_portlet_LoginPortlet_loginForm, #_com_liferay_login_web_portlet_LoginPortlet_fm {
```

Comment:
technically this comma-delimited list of selectors should be alpha sorted as well :(. Sorry, missed it the first time around.

Reply:
@alecsloan this should still be fixed

Reply:
Updated, thanks!